delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/error/TokenError.scala

```diff
@@ -0,0 +1,48 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.error
+
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.contexts
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.ContextValue
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.JsonLdContext.keywords
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.encoder.JsonLdEncoder
+import ch.epfl.bluebrain.nexus.delta.sdk.http.HttpClientError
+import io.circe.syntax.{EncoderOps, KeyOps}
+import io.circe.{DecodingFailure, Encoder, JsonObject}
+
+sealed abstract class TokenError(reason: String) extends SDKError {
```

Comment:
`AuthTokenError` or whatever makes it easier to distinguish from the user tokens who can also raise errors